docs/contributors/getting-started.md

```diff
@@ -233,8 +233,8 @@ sbt
 ## Manual tests
 
 Some functionality is best to manually test through an editor. A common workflow
-while iterating on a new feature is to run `publishLocal` and then open an
-editor in a small demo build.
+while iterating on a new feature is to run `publishLocal;++3.2.2 mtags/publishLocal`(change 3.2.2 to the Scala version you want to test) and then run `Metals:Restart server` for opened projects or open an
```

Comment:
Thanks for adding this @doofin, although if you look right below this we actually specify it in the next paragraph.

Reply:
yeah but it's a bit confusing for new comers,so a one liner might be more friendly?

Reply:
Ah, well I think if we include this here we're just duplicating it again right below that. I actually think what we have down below is more detailed, therefore more helpful. What do you think?

Reply:
Logicallay speaking yes, but when myself try to do manual test first I find it quite confusing at this: 

> It's important to note that sbt publishLocal will create artifacts only for the Scala version currently used in Metals and trying to use the snapshot version with any other Scala version will not work

I'll edit and remove the duplicate stuff about vscode

Reply:
Alright, I ended up changing this up a bit. I still don't want to always mention publishing mtags for a specific version because there are situations where you're working on a feature that doesn't actually rely on needing to do that, so a `publishLocal` is fine. I've clarified that further and put a commit on top of yours.